src/relayserver/RelayServer.ts

```diff
@@ -140,9 +140,14 @@ export class RelayServer extends EventEmitter {
     }
 
     // Check that the gasPrice is initialized & acceptable
-    if (this.gasPrice > parseInt(req.relayRequest.relayData.gasPrice)) {
+    const requestGasPrice = parseInt(req.relayRequest.relayData.gasPrice)
+    if (this.minGasPrice > requestGasPrice) {
       throw new Error(
-        `Unacceptable gasPrice: relayServer's gasPrice:${this.gasPrice} request's gasPrice: ${req.relayRequest.relayData.gasPrice}`)
+        `gasPrice too low: relayServer's gasPrice:${this.minGasPrice} request's gasPrice: ${requestGasPrice}`)
+    }
+    if (parseInt(this.config.maxGasPrice) < requestGasPrice) {
+      throw new Error(
```

Comment:
please
1. update comment
2. update error text messages

Reply:
how about range check, e.g.

`expected gas price X to be in range YYY .. ZZZ`

Reply:
fixed